src/widget/wtracktableview.cpp

```diff
@@ -110,6 +111,11 @@ WTrackTableView::WTrackTableView(QWidget * parent,
 
     connect(this, SIGNAL(scrollValueChanged(int)),
             this, SLOT(slotScrollValueChanged(int)));
+
```

Comment:
it doesn't need to be explicitly deleted. Qt is doing this for us when the parent `this`-pointer is deleted.